server.js

```diff
@@ -0,0 +1,56 @@
+const express = require("express");
+const app = express();
+const port = 9090;
+const fs = require("fs").promises;
+
+app.use(express.json());
+
+//----bringing data-------
+
+// ---------   Createing routes  ------------
+
+// reading All albums
+app.get("/albums", (req, res) => {
+  fs.readFile("./albums.json", "utf-8").then((movie) => {
+    movie = JSON.parse(movie);
+    res.status(200).send(movie);
+  });
+});
+
+// reading albums by id
+app.get(`/albums/:id`, (req, res) => {
+  const id = +req.params.id;
+  const index = fs.readFile("./albums.json", "utf-8").then((movie) => {
+    const searchedAlbum = JSON.parse(movie).findIndex(
+      (album) => +album.id === +req.params.id
```

Comment:
I prefer to be a little more explict with my number conversions, this syntax is a little hard to read quickly, but this is a matter of personal preference